[it] localizzazione termine cloud computing (#617)
cncf/glossary: content/it/cloud-computing.md

---
title: Cloud Computing
status: Completed
category: Concetto
---

## Cos'è

> Il cloud computing è un modello che offre risorse di calcolo come CPU, rete e capacità del disco su richiesta su Internet. Il cloud computing offre agli utenti la possibilità di accedere e utilizzare la potenza di calcolo in una posizione fisica remota. I provider cloud come AWS, GCP, Azure, DigitalOcean e altri offrono tutti a terze parti la possibilità di affittare l'accesso alle risorse di calcolo in più posizioni geografiche.

Reviewer: La prima parte è piuttosto ambigua anche in inglese, sembra accomunare le compute resources a network e disk... Io direi di tradurre così
```suggestion
Il _cloud computing_ è un modello che offre risorse di calcolo (come le CPU) e funzionalità di rete e di disco su richiesta, tramite Internet. Il cloud computing offre agli utenti la possibilità di accedere e utilizzare la potenza di calcolo in una posizione fisica remota. I provider cloud come AWS, GCP, Azure, DigitalOcean e altri offrono tutti a terze parti la possibilità di affittare l'accesso alle risorse di calcolo in più posizioni geografiche.
```

> ## Quale problema affronta 

Reviewer 2: ```suggestion
## Quali problematiche affronta
```

> Le organizzazioni tradizionalmente hanno dovuto affrontare due problemi principali quando tentavano di espandere l'uso della potenza di calcolo. Possono o acquistare, supportare, progettare e pagare strutture per ospitare i loro server fisici e la rete o espandere e mantenere tali strutture. Il cloud computing consente alle organizzazioni di esternalizzare parte delle proprie esigenze informatiche a un'altra organizzazione.

Reviewer: Questa è una proposta. Mi piaceva che dava l'idea della commodity di cui si parla nel cloud computing, come per un'azienda è il fabbisogno energetico...
```suggestion
Le organizzazioni tradizionalmente hanno dovuto affrontare due problemi principali quando tentavano di espandere l'uso della potenza di calcolo. Possono o acquistare, supportare, progettare e pagare strutture per ospitare i loro server fisici e la rete; oppure espandere e mantenere tali strutture. Il cloud computing consente alle organizzazioni di esternalizzare parte del proprio fabbisogno di computazione a un'altra organizzazione.
```

Reviewer 3: @sistella bella proposta!

## In che modo aiuta

> I provider cloud offrono alle organizzazioni la possibilità di noleggiare risorse di calcolo su richiesta e pagare per l'utilizzo. Ciò consente due innovazioni principali: le organizzazioni possono provare le risorse senza perdere tempo a pianificare e spendere denaro o risorse per la nuova infrastruttura fisica e possono [scalare](/scalabilità/) secondo necessità e su richiesta. Il cloud computing consente alle organizzazioni di adottare tutta l'infrastruttura di cui hanno bisogno.

Reviewer: eh, io ci provo sempre :D. Ho tolto il link perchè non abbiamo ancora tradotto scalability.
ho aggiunto 'e sola' perchè ho visto che in inglese dice 'to adopt as much **or as little** infrastructure ...'
```suggestion
I provider cloud offrono alle organizzazioni la possibilità di noleggiare risorse di calcolo su richiesta e pagare per l'utilizzo. Ciò consente due innovazioni principali: le organizzazioni possono provare le risorse senza perdere tempo a pianificare e spendere denaro o risorse per la nuova infrastruttura fisica; inoltre possono essere scalabili secondo necessità e su richiesta. Il cloud computing consente alle organizzazioni di adottare tutta e sola l'infrastruttura di cui hanno bisogno.
```

Reviewer 3: direi, però, "adottare l'infrastruttura necessaria, tanta o poca che sia."

Reviewer: incluso nel suggerimento

> I provider cloud offrono alle organizzazioni la possibilità di noleggiare risorse di calcolo su richiesta e pagare per l'utilizzo. Ciò consente due innovazioni principali: le organizzazioni possono provare le risorse senza perdere tempo a pianificare e spendere denaro o risorse per la nuova infrastruttura fisica; inoltre possono essere scalabili secondo necessità e su richiesta. Il cloud computing consente alle organizzazioni di adottare l'infrastruttura necessaria, tanta o poca che sia.

Reviewer 2: ```suggestion
I provider cloud offrono alle organizzazioni la possibilità di noleggiare risorse di calcolo su richiesta e pagare per l'utilizzo. Ciò consente due innovazioni principali: le organizzazioni possono provare le risorse senza perdere tempo a pianificare e spendere denaro o risorse per la nuova infrastruttura fisica; inoltre possono essere [scalabili](/it/scalability/) secondo necessità e su richiesta. Il cloud computing consente alle organizzazioni di adottare l'infrastruttura necessaria, tanta o poca che sia.
```

Reviewer 2: @sistella
